src/backend/api/Fusion.Resources.Domain/Queries/GetDepartmentPersonnel.cs

```diff
@@ -23,7 +23,7 @@ public GetDepartmentPersonnel(string department, ODataQueryParams? queryParams =
             QueryParams = queryParams;
         }
 
-
+        public bool IncludeSubdepartments { get; set; } = false;
```

Comment:
..SubDepartment?

Reply:
According to Merriam Webster subdepartment is the most common spelling. https://www.merriam-webster.com/dictionary/subdepartment